### ISSUE TITLE
Issue highlighting not removed when issue deleted

### ISSUE THREAD

**Issue by bengodden** [kiyanaw/kiyanaw-backend#70]
When deleting an issue, the highlighting remains even when the issue no longer exists.

See example:  https://app.kiyanaw.net/transcribe-edit/4626c2a0/wavesurfer_b6kuqb2gh4

**Comment by aaronfay** [comment]
Yup, I've seen that too. For the time being you can "Resolve" the issue first, and then delete, that will remove highlighting as a workaround.
